File: AoC2023/day_02_02/java/test/java/solve/SolverTest.java
package solve;

import static org.junit.Assert.assertEquals;

import org.junit.Test;
import org.junit.jupiter.params.ParameterizedTest;
import org.junit.jupiter.params.provider.ValueSource;

public class SolverTest {

  @Test
  public void testSolver() {
    Utils utils = new Utils();
    String content = utils.getResourceFileAsString("input.txt");
    assertEquals(Solver.solve(content), "2286");
  }
}

File: AoC2023/day_04_02/kotlin/test/kotlin/solve/SolverTest.kt
package solve

import kotlin.test.Test
import kotlin.test.assertEquals

internal class SolverTest {

    @Test
    fun testHelloWorld() {
        val utils: Utils = Utils()
        val content = utils.getResourceFileAsString("input.txt")
        assertEquals(Solver.solve(content), "30")
    }
}
